packages/frontend/src/widgets/short-info/ui/select-guests-button/SelectGuestsButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { ButtonHTMLAttributes, DetailedHTMLProps, ReactNode } from 'react'
+import classNames from 'classnames'
+
+import styles from './SelectGuestsButton.module.scss'
+
+interface ISelectGuestsButton extends DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
+  fill: boolean
+  children: ReactNode
+  isActive: boolean
+}
+
+export const SelectGuestsButton = ({ fill = false, onClick, children, isActive }: ISelectGuestsButton): JSX.Element => (
+  <button
+    className={ classNames(styles.button, {
+      [styles['button-filled']]: fill,
+      [styles['button-active']]: isActive
+    }) }
+    onClick={ onClick }
+  >
+    {children}
+  </button>
```

Comment:
Нужно собрать все, что будет приходить сверху в пропс и потом прокинуть их на кнопку

```suggestion
export const SelectGuestsButton = ({ fill = false, children, isActive, ...props }: ISelectGuestsButton): JSX.Element => (
  <button
    className={ classNames(styles.button, {
      [styles['button-filled']]: fill,
      [styles['button-active']]: isActive
    }) }
    {...props}
  >
    {children}
  </button>
```

Reply:
Поправил